src/js/gallery/service/resize_emulator/action.coffee

```diff
@@ -0,0 +1,13 @@
+angular.module('cycleGallery').service 'ResizeEmulatorAction', [
+  ->
+
+    class ResizeEmulatorAction
+
+      constructor : (fn, key)->
```

Comment:
Colon assignment without proper spacing<br>Function arrow (->) must be spaced properly